conf/flight_plans/tudelft/rotating_wing_EHVB.xml

```diff
@@ -69,15 +69,21 @@
     </block>
     <block name="Start Engine" strip_button="Start Engines" strip_icon="on.png">
         <call_once fun="rotwing_request_configuration(ROTWING_CONFIGURATION_HOVER)"/>
+        <call_once fun="nav_set_heading_current()"/>
         <call_once fun="NavResurrect()"/>
         <attitude pitch="0" roll="0" throttle="0" until="stage_time>10" vmode="throttle"/>
     </block>
+    <block name="Wait takeoff">
+      <exception cond="stage_time > 10" deroute="Holding point"/>
```

Comment:
you should avoid using `>' characters. Use `@GT` or `@DEREF` if needed. See https://wiki.paparazziuav.org/wiki/Flight_Plans#Expressions